src/components/Login/Login.js

```diff
@@ -1,5 +1,126 @@
-import React from "react";
-
+import React, { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import { logInHandler } from "../../ApiServices/ApiServices";
+import { useAuth } from "../../Context/auth-context";
 export default function Login() {
-  return <div>Login</div>;
+  const [login, setLogin] = useState({
+    email: "",
+    password: "",
+  });
+
+  const { authstate, setAuthState } = useAuth();
+  const navigateTo = useNavigate();
+  const guestCredential = {
+    email: "adarshbalika@gmail.com",
+    password: "adarshBalika123",
+  };
```

Comment:
Can shift these guest credentials to a separate file called as constants.js and then use it from there making it easy to understand!!

Reply:
Okay